### PR TITLE
feat(unity): Added Unity to screenshots

### PR DESCRIPTION
**Pull request by bitsandfoxes**

### CODE REVIEW

File: src/includes/enriching-events/attach-screenshots/unity.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+The **Attach Screenshot** options can be found on the `Enrichment` tab in the Editor Configuration Window.
```
Comment (imatwawana) [inline src/includes/enriching-events/attach-screenshots/unity.mdx:1]:
```suggestion
The "Attach Screenshot" options are on the "Enrichment" tab of the editor configuration window.
```

File: src/includes/enriching-events/attach-screenshots/unity.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+![Screenshots Thumbnail Example](unity-screenshots.png)
+
+<Note>
+
+Note that capturing a screenshot in the Unity SDK requires running on the main thread.
```
Comment (imatwawana) [inline src/includes/enriching-events/attach-screenshots/unity.mdx:7]: Making a wording change here to try to clarify, but if I've done so incorrectly, let me know.

```suggestion
Capturing a screenshot in Unity requires running the SDK on the main thread.
```

File: src/includes/enriching-events/attach-screenshots/unity.mdx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+</Note>
+
+### Max Width and Max Height
+
+The default value of 0 means that screenshots get captured in the game's current [Screen](https://docs.unity3d.com/ScriptReference/Screen.html) size. With max values greater than 0 but less than the screen size, the SDK will try to scale down the captured screenshot while preserving the apect ratio. This allows for a smaller file being sent.
```
Comment (imatwawana) [inline src/includes/enriching-events/attach-screenshots/unity.mdx:13]: I've made a change to the last sentence that I'm hoping doesn't change the meaning too much. If it does, lemme know and I'll try to edit a different way.
```suggestion
The default value of 0 means that screenshots are captured in the game's current [screen](https://docs.unity3d.com/ScriptReference/Screen.html) size. If max values are greater than 0 but less than the screen size, the SDK will try to scale down the captured screenshot while preserving the aspect ratio. This allows for a smaller file to be generated.
```

**Review by imatwawana — COMMENTED** [review]
I've made some changes that might change meaning so if there's anything here that doesn't work, let me know.

File: src/includes/enriching-events/attach-screenshots/android.mdx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+You can enable it as shown below:
```
Comment (marandaneto) [inline src/includes/enriching-events/attach-screenshots/android.mdx:1]:
```suggestion
You can enable it as shown below:

```

File: src/includes/enriching-events/attach-screenshots/unity.mdx [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+![Screenshots Thumbnail Example](unity-screenshots.png)
+
+<Note>
+
+Note that capturing a screenshot in the Unity SDK requires running on the main thread.
+
+</Note>
+
+### Max Width and Max Height
```
Comment (marandaneto) [inline src/includes/enriching-events/attach-screenshots/unity.mdx:11]: Please amend the specs with this 2 configs https://develop.sentry.dev/sdk/features/#screenshots

File: src/includes/enriching-events/attach-screenshots/unity.mdx (hunk 2 above)
Comment (marandaneto) [inline src/includes/enriching-events/attach-screenshots/unity.mdx:7]: This is important to be written in the common docs as well (src/platforms/common/enriching-events/screenshots.mdx)

File: src/includes/enriching-events/attach-screenshots/unity.mdx (hunk 5 above)
Comment (marandaneto) [reply src/includes/enriching-events/attach-screenshots/unity.mdx:11]: I'd also add this new 2 options in here below https://docs.sentry.io/platforms/android/configuration/options/#attach-screenshot

**Comment by marandaneto** [comment]
https://docs.sentry.io/platforms/android/configuration/options/#attach-screenshot should be available for Unity as well.

**Pull request closed without merging**